uint8[4] id
float64 lat
float64 longi
float64 elev
uint8 transmission
uint16 speed
uint16 heading
int16 angle
int16 acc_long
int16 acc_lat
uint8 acc_vert
int16 acc_yaw
uint8 wheelbrakes
uint8 traction
uint8 abs
uint8 scs
uint8 brakeboost
uint8 auxbrakes
int32 width
int32 length
